provider/oci/environ.go

```diff
@@ -645,8 +656,9 @@ func (e *Environ) StartInstance(ctx envcontext.ProviderCallContext, args environ
 	}
 
 	result := &environs.StartInstanceResult{
-		Instance: instance,
-		Hardware: instance.hardwareCharacteristics(),
+		DisplayName: displayName,
```

Comment:
also needs a test

Reply:
Turns out that this is much harder to test than I thought it would be. The OCI provider doesn't seem to actually test `StartInstance`. All of the current tests appear to focus on interactions with the upstream OCI library.